docs/understanding-radicle/faq.md

```diff
@@ -1,14 +1,135 @@
 ---
 id: faq
-title: FAQ
+title: Frequently Asked Questions
 ---
+## How do I get started?
+Head over to the [Getting Started][gs] section for instructions on how to
+download and install the Radicle Upstream client.
 
-Why can't I git clone a Radicle project?
-How is Radicle more secure than GitHub?
-How is collaborating on Radicle different than GitHub? [social model?]
-What is a remote?
-How does Radicle interact with Git?
-- https://www.git-scm.com/docs/gitremote-helpers
-What does following mean in Radicle?
-How will issues and prs work? 
-Why no CLI?
\ No newline at end of file
+## How is collaborating on Radicle different than GitHub?
+In contrast to centralized code collaboration platforms, Radicle is designed for
+bazaar-style collaboration. On the Radicle network, content is distributed
+peer-to-peer via a process called gossip. This means that peers are in control
+of their social interactions as they self-host their own content and the content
+of any peers they are interested in. This also means that within projects, there
+isn't a single `master` branch that contributors merge into. Each peer maintains
+a view of a project with their changesets and branches. These views are gossiped
+around to other peers that are interested in those changes. Read more about the
+implications and approach to this design [here][sm].
+
+## How is Radicle more secure than centralized platforms?
```

Comment:
Is `secure` the word here? Maybe this is more about `How is radicle different form centralised code collaboration stacks?` And then it goes before this question: `How is collaborating on Radicle different than GitHub?`

Reply:
Hm, I guess it's nice to call out how Radicle is different from a security context... Similar to that Security Risks and Solutions @cloudhead put together that one time.